TypeScript.YAML-tmLanguage

```diff
@@ -176,7 +176,7 @@ repository:
     beginCaptures:
       '1': { name: storage.type.function.ts }
       '2': { name: entity.name.function.ts }
-    end: (?=;|\})|(?<=\})
+    end: (?=;|\}|\n)|(?<=\})
```

Comment:
As discussed, this may not work for:
```javascript
function
foo() {
}
```
But since this is rarely used, I think it should be fine.